vmdb/app/assets/javascripts/miq_dhtmlxgrid.js

```diff
@@ -101,46 +101,46 @@ function miqInitGrids() {
 
 // Initialize a single grid (is called directly after an AJAX trans)
 function miqInitGrid(grid_name) {
-  var grid_hash = miq_grids[grid_name] // Get the hash for the passed in grid
+  var grid_hash = miq_grids[grid_name]; // Get the hash for the passed in grid
   var miq_grid_checks = ""; // Keep track of the grid checkboxes
 
   // Build the grid object, then point a local var at it
-  eval(grid_name + " = new dhtmlXGridObject('" + grid_hash['g_id'] + "')");
+  eval(grid_name + " = new dhtmlXGridObject('" + grid_hash.g_id + "')");
```

Comment:
Evil :). Maybe `window[grid_name] = new dhtmlXGridObject(grid_hash.g_id);` instead? At least until we can figure out who expects the variable to exist and move it under .. `ManageIQ.grids[grid_name]` or somewhere.